code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -1,13 +1,14 @@
 /turf
 	//used for temperature calculations
-	var/thermal_conductivity = 0.05
+	var/thermal_conductivity = 0.005
```

Comment:
Is this from putnams superconduction revival?

Reply:
maybe, will confirm